packages/react-native-renderer/src/ReactNativeFiberInspector.js

```diff
@@ -89,6 +91,34 @@ if (__DEV__) {
     }));
   };
 
+  const getInspectorDataForInstance = function(closestInstance, frame): Object {
```

Comment:
Since this is returning Object, you won't get type safety that `getInspectorDataForViewAtPoint` is giving you `TouchedViewDataAtPoint`

Reply:
Nice catch, fixed.